docs/en/source/understanding/meet.rst

```diff
@@ -0,0 +1,82 @@
+
+2. Complete availability
+===============================
+
+The content of the gazettes , previously locked in PDFs, must be converted into plain text and stored in an open TXT format. Both the original and the open files are made fully available by Querido Diário.
+
+3. Huge searchable database
+======================================
+
+By gradually adding more of the 5,570 Brazilian municipalities, we aim to build a vast and pioneering database on local public administration, enabling the Brazilian society to benefit from this information. To achieve this, we will provide extensive search resources for one or multiple municipalities simultaneously.
+
+4. Free access
+===================
+
+Access to the data stored by Querido Diário and research resources implemented should be free for users in a friendly and timely manner, and for developers or machines, programmatically.
+
+Facing the Data Desert
+*********************************
+
+To realize its pillars, Querido Diário must face various challenges, all related to the characteristics of publishing municipal gazettes in a context where there is no national legislation standardizing it. In this section, we outline the most relevant aspects that constitute obstacles being overcome by the project.
+
+1. Electronic publication
+===========================
+
+For Querido Diário to have access to a municipality's gazettes , the municipality must publish files on a *website* publicly available on the *Internet*. A municipality that does no have an official *website* or does not make its gazettes available online cannot be integrated into the project.
+
+2. File format
+=======================
+
+When we say "obtain official Gazettes," we are specifically interested in their content: it is the text of the decree, the ordinance, the bidding, etc, that we want; these are stored and shared through files.
+
+Some file formats are more visual and suitable for human reading, while others prioritize easy access to their textual content, making them suitable for machine reading in general, municipalities prioritize publication for human reading and provide their official acts through PDF files, making automatic content processing difficult. Moreover, they can be of two types:
+
+- **Text PDF**: the content of the PDF is text, probably generated directly by the text editor program.
+
+- **Image PDF**: the content of the PDF is an image, and the file was created by scanning the printed edition.
+
+.. _tipo-diarios:
+
+3. Type of Official Gazette
+===========================
+
+Despite the lack of a legal standard, we find it possible to classify electronic gazettes into three categories:
+
+- **Aggregated Gazette**: the document contains official acts from various municipalities, often because they are part of an Association/Federation or the Diário do Estado(State Gazette) has a section for municipalities. An example is the  `Diário Oficial da Associação de Municípios de Alagoas`_.
```

Comment:
```suggestion
- **Aggregated Gazette**: the document contains official acts from various municipalities, often because they are part of an Association/Federation or the State Gazette has a section for its municipalities. An example is the  `Municipalities Association of Alagoas State Official Gazette`_.
```